src/client/components/features/JeDeviensMentor/JeDeviensMentor.module.scss

```diff
@@ -0,0 +1,2 @@
+@import "@styles/utilities";
```

Comment:
Pourquoi garder un fichier vide ?

Reply:
juste in case, mais on est d'accord que c'est nul, je supprime